src/main/java/uk/gov/hmcts/reform/civil/controllers/fees/ServiceRequestUpdateClaimIssuedCallbackController.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.civil.controllers.fees;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.civil.enums.FeeType;
+import uk.gov.hmcts.reform.civil.model.ServiceRequestUpdateDto;
+import uk.gov.hmcts.reform.civil.service.PaymentRequestUpdateCallbackService;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class ServiceRequestUpdateClaimIssuedCallbackController {
+
+    private final PaymentRequestUpdateCallbackService requestUpdateCallbackService;
+
+    @PutMapping(path = "/service-request-update-claim-issued", consumes = APPLICATION_JSON, produces = APPLICATION_JSON)
+    @ApiOperation(value = "Ways to pay will call this API and send the status of payment with other details")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Callback processed.", response = CallbackResponse.class),
+        @ApiResponse(code = 400, message = "Bad Request")})
+    public void serviceRequestUpdate(@RequestBody ServiceRequestUpdateDto serviceRequestUpdateDto) {
+        try {
+            requestUpdateCallbackService.processCallback(serviceRequestUpdateDto, FeeType.CLAIMISSUED.name());
+        } catch (Exception ex) {
+            log.error(
+                "Payment callback is unsuccessful for the CaseID: {}",
+                serviceRequestUpdateDto.getCcdCaseNumber()
+            );
+        }
```

Comment:
Don't swallow exceptions.

Remove the try/catch statement altogether. Let the exception be thrown.

Reply:
We're still swallowing this exception, because the tests break when we do not. I have tried to change tests to reflect it but to no avail. The other endpoint is built similarly (as are the ones in General Applications), so might be worth to raise a ticket that changes all of them - resolved myself, as you are still off on sick leave.